tests/infra/ccf.py

```diff
@@ -238,7 +239,10 @@ def create_and_add_pending_node(self, lib_name, host, args, target_node=None):
             self.consortium.wait_for_node_to_exist_in_store(
                 primary,
                 new_node.node_id,
-                (
+                # When the service is open, it takes a joining node at least 2 join
+                # attempts to retrieve the network secrets
+                timeout=int(args.join_timer * 3 / 1000),
```

Comment:
Why `int`? Assuming this is eventually just passed to `time.sleep` or equivalent, a float is fine. If we're rounding to whole seconds for a reason, we probably want to round up with `math.ceil` (join_timer == 999ms => wait 3 seconds).

Reply:
We actually pass this value to a loop (`for _ in range(timeout):`) hence the integer value. `math.ceil()` is better though 👍